docs/MODULES.md

```diff
@@ -427,7 +421,7 @@ where `options` is an object with the following properties:
 
 ## paint-bucket-module
 
-This module helps you to fill color in pixels
+This is a flood fill module and it helps you to fill color in pixels
```

Comment:
What this module actually does is that it fills a polygonal shape (like a square) with the specified color.